lib/dotiw/action_view/helpers/date_helper.rb

```diff
@@ -6,11 +6,11 @@ module DateHelper
       alias_method :_distance_of_time_in_words, :distance_of_time_in_words
       alias_method :_time_ago_in_words, :time_ago_in_words
 
-      include DOTIW::Methods
```

Comment:
Note that after this we have the new `distance_of_time_in_words`. We could do this again:

```ruby
alias_method :_dotiw_distance_of_time_in_words, :distance_of_time_in_words

# now you can call _dotiw_distance_of_time_in_words instead of super
```

I'd still prefer we find a way to directly invoke it instead of having to alias it again.

Reply:
This file monkey-patches `ActionView::Helpers::DateHelper` and replaces the methods in it to invoke `DOTIW::Methods.distance_of_time_in_words`. If you remove this, then this will no longer work when you just include `ActionView::Helpers`. The reason why it still works in your specs is that you moved out `include DOTIW::Methods` to always be included like so:

```ruby
describe 'A better distance_of_time_in_words' do
  include DOTIW::Methods
...
```

But in an actual application that `include` is now gone and none of the helpers will work.

Reply:
Restored the include in the monkey patch: https://github.com/radar/distance_of_time_in_words/pull/111/commits/13fa3a8a58bb45087b699eef4708aa5f902a4da3